src/storage/src/source/sql_server/replication.rs

```diff
@@ -221,22 +222,51 @@ pub(crate) fn render<G: Scope<Timestamp = Lsn>>(
 
             // Start replicating from the LSN __after__ we took the snapshot.
             let replication_start_lsn = snapshot_lsn.increment();
+            let mut rewinds: BTreeMap<_, _> = export_ids_to_snapshot
+                .values()
+                .flat_map(|export_ids| export_ids.iter().map(|idx| (*idx, replication_start_lsn)))
```

Comment:
hm, I was expecting to see the initial_lsn mentioned here. Don't we want to rewind to that point instead of whatever lsn replication happens to start from?